llama_stack/providers/remote/inference/ollama/ollama.py

```diff
@@ -293,9 +293,7 @@ async def register_model(self, model: Model) -> Model:
         if model.model_type == ModelType.embedding:
             logger.info(f"Pulling embedding model `{model.provider_resource_id}` if necessary...")
             await self.client.pull(model.provider_resource_id)
-            response = await self.client.list()
-        else:
-            response = await self.client.ps()
+        response = await self.client.list()
```

Comment:
self.client.ps() is essential because, when the server starts, the inference endpoint must be actively serving the model. If we use list() instead, we can only confirm that the model exists, but not that it is actually being served. This was previously discussed a few months ago here: https://github.com/meta-llama/llama-stack/pull/403.

Reply:
please try it out w/ `ollama serve`